service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -349,10 +349,6 @@ void setupFunctionalChecks() throws Exception {
             });
 
     doReturn(mockUser).when(samService).getSamUser();
-
-    // setup Sam permission check to return true
-    doReturn(true).when(samService).hasReadPermission();
-    doReturn(true).when(samService).hasComputePermission();
```

Comment:
these upfront mocks were causing some issues in subsequent tests that overwrite the return values - I thought it would be clearer to remove them